setup.py

```diff
@@ -10,7 +10,7 @@
     install_requires=[
         "backoff",
         "celery>=5",
-        "gemlock_parser",
+        "gemlock_parser @ git+https://github.com/containerbuildsystem/gemlock-parser.git@master#egg=gemlock_parser",
```

Comment:
@tichavskym @chmeliik Shall we have this on PyPI?

Reply:
I could definitely upload it there, I just don't know whether we have some login as an organization or how shall I proceed with this?

Reply:
I'd vote against it, the gemlock-parser repo is supposed to be a temporary workaround until the relevant part of the scancode library (where we vendored the code from) is made standalone

https://github.com/containerbuildsystem/cachito/pull/666#issuecomment-1145756553

Reply:
okay, then keep it as is in this PR